artiq/frontend/artiq_master.py

```diff
@@ -133,12 +138,16 @@ def ccb_issue(service, *args, **kwargs):
     })
     experiment_db.scan_repository_async(loop=loop)
 
+    master = MasterProcess()
+    setattr(MasterProcess, 'terminate', lambda self: signal_handler_task.cancel())
```

Comment:
Ah, I see now. Feels like a workaround...

Reply:
Well there is a way to add a method to SimpleNamespace, but it would it to the SimpleNamespace as class and affect all other instances.

Reply:
In that case, what about just using `master = object()`?